packages/appium/docs/mkdocs-cn.yml

```diff
@@ -0,0 +1,54 @@
+INHERIT: ./base-mkdocs.yml
+site_url: https://appium.github.io/appium/docs/en
```

Comment:
this (and the line below) should probably reference `cn` rather than `en`, right?

Reply:
Thanks for pointing that, try to local build, met below exception
`jinja2.exceptions.TemplateNotFound: partials/languages/cn.html`
Just search from mkdocs, seems they won't support such word, so change as zh and reformat all folder and resources.
https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language